group_vars/gxconfig.yml

```diff
@@ -1130,7 +1185,7 @@ base_app_main: &BASE_APP_MAIN
   # via a proxy server.  These options should be self explanatory and so
   # are not documented individually.  You can use these paths (or ones
   # in the proxy server) to point to your own styles.
-  static_enabled: true
+  #static_enabled: true
```

Comment:
it should be the default value

Reply:
It is the [default value](https://github.com/galaxyproject/galaxy/blob/release_24.1/lib/galaxy/config/schemas/config_schema.yml#L1768)